src/usb.c

```diff
@@ -2,19 +2,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <stdint.h>
+#include <limits.h>
```

Comment:
Where is this include used in `usb.c`?

Reply:
Try to resolve this https://github.com/texane/stlink/issues/897#issuecomment-604588214

Reply:
Ah, thx :+1: